src/main/java/uk/gov/justice/hmpps/datacompliance/repository/jpa/model/referral/ReferralResolution.java

```diff
@@ -52,7 +53,7 @@
     private Long resolutionId;
 
     @NotNull
-    @OneToOne(fetch = LAZY)
+    @OneToOne(fetch = EAGER)
```

Comment:
Some unrelated changes to fetch eager since these entities are all immediately linked by their `getOffenderNumber` method anyway.